src/datasources/cache/__tests__/fake.cache.service.spec.ts

```diff
@@ -16,7 +16,7 @@ describe('FakeCacheService', () => {
     );
     const value = faker.string.alphanumeric();
 
-    await target.set(cacheDir, value, 0);
+    await target.set(cacheDir, value, faker.number.int());
```

Comment:
If `0` is given as the `expireTimeSeconds`, it [shouldn't set the key](https://github.com/safe-global/safe-client-gateway/blob/80134e6ff4bf3dc190988b3485db3803a76d5bd3/src/datasources/cache/__tests__/fake.cache.service.ts#L45-L47) and `faker.number.int` [can return `0`](https://fakerjs.dev/api/number.html#int). We should use a `min` of `1` for these examples.